infra-parent/infra-core/src/main/java/com/redhat/service/smartevents/infra/core/models/responses/ListResponse.java

```diff
@@ -14,7 +14,7 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public abstract class ListResponse<T> {
 
-    public static <T, V> ListResponse<V> fill(ListResult<T> source, ListResponse<V> target, Function<T, V> converter) {
+    public static <S, V> ListResponse<V> fill(ListResult<S> source, ListResponse<V> target, Function<S, V> converter) {
```

Comment:
Not really related to this PR: just a small readability improvement changing the Generic of the static method